### PR TITLE
Add CI

### PR DESCRIPTION
**Pull request by ruioliveira02**

### CODE REVIEW

File: .github/workflows/style.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  style:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: ['18.12.0']
+
+    steps:
+      - name: ☁ Checkout code
+        uses: actions/checkout@v2
```
Comment (ruilopesm) [inline .github/workflows/style.yml]:
```suggestion
        uses: actions/checkout@v3
```

File: .github/workflows/style.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+      - name: 📬 Caching
+        uses: actions/cache@v2
```
Comment (ruilopesm) [inline .github/workflows/style.yml]:
```suggestion
        uses: actions/cache@v3
```

File: .github/workflows/style.yml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+        with:
+          path: |
+            **/node_modules
+            ${{ github.workspace }}/.next/cache
+          key: ${{ runner.os }}-modules-${{ hashFiles('**/package-lock.json') }}
+
+      - name: 🟩 Setup node
+        uses: actions/setup-node@v2
```
Comment (ruilopesm) [inline .github/workflows/style.yml]:
```suggestion
        uses: actions/setup-node@v3
```

File: .github/workflows/style.yml [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+        with:
+          node-version: ${{ matrix.node }}
+
+      - uses: actions/cache@v2
```
Comment (ruilopesm) [inline .github/workflows/style.yml]:
```suggestion
      - uses: actions/cache@v3
```

File: .gitignore [hunk 5]
```diff
@@ -27,6 +27,7 @@ yarn-error.log*
 
 # local env files
 .env*.local
+.env
```
Comment (ruilopesm) [inline .gitignore:30]:
```suggestion
.env.*
!.env.*.sample
```

Comment (ruioliveira02) [reply .gitignore:30]: I will add those lines. However, none of them match `.env`, so I will keep it

**Review by ruilopesm — APPROVED** [review]

**Pull request merged**